ckanapi/localckan.py

```diff
@@ -44,7 +48,45 @@ def call_action(self, action, data_dict=None, context=None, apikey=None,
             raise CKANAPIError("LocalCKAN.call_action does not support "
                 "use of apikey parameter, use context['user'] instead")
         if files:
-            raise CKANAPIError("TestAppCKAN.call_action does not support "
-                "file uploads, consider contributing it if you need it")
+            return self._handle_files(action, data_dict, context, files)
+
         # copy dicts because actions may modify the dicts they are passed
         return self._get_action(action)(dict(context), dict(data_dict))
+
+    def _handle_files(self, action, data_dict, context, files):
+        if action not in ['resource_create', 'resource_update']:
+            raise CKANAPIError("LocalCKAN.call_action only supports file uploads for resources.")
```

Comment:
it should work the same way for uploading images for groups and orgs (and any other actions that decide to handle file uploads) Would you remove this check?